server/rules-engine/enums.js

```diff
@@ -0,0 +1,15 @@
+const rulesTypes = {
+  eligibility: 'eligibility',
+  fact: 'fact'
+}
```

Comment:
one little trick I use for non-extensible objects is to wrap it in Object.freeze (e.g. `Object.freeze({ eligibility: 'eligibility', fact: 'fact' })`). This covers the sometimes opaque behaviour of const, where you can extend and change the object so long as you don't assign a new object to the variable.

Reply:
Nice.